src/components/Feed/__snapshots__/Feed.test.js.snap

```diff
@@ -39,7 +39,7 @@ Array [
       >
         <img
           className="avatar img-fluid rounded-circle"
-          src="avatar.jpg"
+          src={Object {}}
```

Comment:
Why did this happened?

Reply:
i didnt succeed to use the static file on the test. I dont know if I couldnt or if its a thing we should always do testing, but i mocked the static and style files here:   https://github.com/isaacbatst/twitter-gemini-frontend/blob/65b7ab2fe5ce6da34a9bcbf3089a65cf84651798/jest.config.json#L6

and here: 
https://github.com/isaacbatst/twitter-gemini-frontend/blob/65b7ab2fe5ce6da34a9bcbf3089a65cf84651798/jest.config.json#L7

with just a empty object. So the image became an empty object, is this a nice way to do it?

Reply:
i mean, to do tests envolving static and style files?

Reply:
Ahh, sorry, I understand now.